timers.js

```diff
@@ -1,4 +1,36 @@
+var resetB=document.querySelector('#reset');
+var startB=document.querySelector('#start');
+var pauseB=document.querySelector('#pause');
+var seconds;
+var timerId;
 
+resetB.addEventListener("click", resetTimer);
+startB.addEventListener("click", startTimer);
+pauseB.addEventListener("click", stopTiming);
+function startTimer(){
+  startB.removeEventListener('click', clearInterval(timerId));
```

Comment:
Look at [removeEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener). The removeEventListener isn't actually doing anything here. If we change this line to `clearInterval(timerId)` it will work just the same